docker/README.md

```diff
@@ -0,0 +1,83 @@
+You can use Docker to easily try out or test sendgrid-python.
+
+<a name="Quickstart"></a>
+# Quickstart
+
+1. Install Docker on your machine.
+2. Run `docker run -it sendgrid/sendgrid-python`.
+
+<a name="Info"></a>
+# Info
+
+This Docker image contains
+ - `sendgrid-python` and `python-http-client`
+ - Stoplight's Prism, which lets you try out the API without actually sending email
+ - A complete setup for testing the repository or your own fork
+
```

Comment:
It's a little unwieldy to mention the specific versions (especially as this can change); I'll mention details of the setup here and save individual versions for running the container.